include/simdjson/dom/parser.h

```diff
@@ -347,24 +333,13 @@ class parser {
   /** @private Use simdjson_error instead */
   using InvalidJSON [[deprecated("Use simdjson_error instead")]] = simdjson_error;
 
-  /** @private Next location to write to in the tape */
-  uint32_t current_loc{0};
-
-  /** @private Number of structural indices passed from stage 1 to stage 2 */
-  uint32_t n_structural_indexes{0};
-  /** @private Structural indices passed from stage 1 to stage 2 */
-  std::unique_ptr<uint32_t[]> structural_indexes{};
-
-  /** @private Tape location of each open { or [ */
-  std::unique_ptr<internal::scope_descriptor[]> containing_scope{};
-
-  /** @private Return address of each open { or [ */
-  std::unique_ptr<internal::ret_address[]> ret_address{};
+  /** @private [for benchmarking access] The implementation to use */
+  std::unique_ptr<internal::dom_parser_implementation> implementation{};
 
   /** @private Use `if (parser.parse(...).error())` instead */
-  bool valid{false};
+  bool valid;
```

Comment:
The explicit initializer values that you are removing might trigger new warnings in some static analyzers.

Reply:
Ah. I don't really mind reinstating them, I just want to remove them really badly (soon it will be possible :)), and subconsciously minimize their impact whenever I notice them.

Reply:
Reinstated.